breathecode/services/seo/actions/_2_internal_linking.py

```diff
@@ -50,7 +54,7 @@ def internal_linking(client, report):
         # clusters must be linked
         path = url.path
         if path == '': path = href
-        missing_cluster_paths = [i for i in missing_cluster_paths if i != path]
+        missing_cluster_paths = [i for i in missing_cluster_paths if i.lower() != path.lower()]
```

Comment:
if the last character of the link has a forward slask, remove it before comparing (from both strings).